src/pages/auth/Login.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
+import './login.css';
+import { useNavigate } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { clearError } from '../../states/slice/authSlice';
+import { authData } from '../../network/api-crud';
+import Sidebar from '../../components/sidebar/Sidebar';
+
+export default function Login() {
+
+    const [userName, setUserName] = useState('kminchelle');
```

Comment:
initial state should be an empty string for username and password

Reply:
done